src/main/proto/ga4gh/reads.proto

```diff
@@ -151,13 +154,19 @@ message ReadAlignment {
   // The read ordinal in the fragment, 0-based and less than numberReads. This
   // field replaces SAM flag 0x40 and 0x80 and is intended to more cleanly
   // represent multiple reads per fragment.
+  // A read_number of 0 represents SAM flag 0x40 being set and 0x80 being
+  // unset.
+  // A read_number of 1 represents SAM flag 0x80 being set and 0x40 
+  // being unset.
+  // A read_number other than 0 or 1 means both SAM flags 0x40 and 0x80 are 
+  // set.
   int32 read_number = 8;
 
   // The read fails platform or vendor quality checks (SAM flag 0x200).
   bool failed_vendor_quality_checks = 9;
 
-  // The alignment for this alignment message. This field will be null if the
-  // read is unmapped.
+  // The alignment for this alignment message. This field will be zero if the
```

Comment:
Saying the field will be `0` confused me. I think you mean to say if it isn't set perhaps?